_rules/aria-state-or-property-permitted-5c01ea.md

```diff
@@ -49,81 +50,83 @@ _There are no major accessibility support issues known for this rule._
 
 #### Passed Example 1
 
-`aria-pressed` state is supported for role `button`.
+The `aria-pressed` [state][] is [supported][] for the [semantic role][] `button`, which is the [implicit role][] for `button` elements.
 
 ```html
-<div role="button" aria-pressed="false">My button</div>
+<button aria-pressed="false">My button</button>
 ```
 
 #### Passed Example 2
 
-`aria-pressed` state is supported for `role=button` that is the [implicit role](#implicit-role) for `button` element.
+The `aria-pressed` [state][] is [supported][] for the [semantic role][] `button`, which is the [explicit role][] of this `div` element.
 
 ```html
-<button aria-pressed="false">My button</button>
+<div role="button" aria-pressed="false">My button</div>
 ```
 
 #### Passed Example 3
 
-Global state that is supported by all base markup elements.
+The `aria-busy` [state][] is a [global][] [state][] that is [supported][] by all elements, even without any [semantic role][].
 
 ```html
 <div aria-busy="true">My busy button</div>
 ```
 
 #### Passed Example 4
 
-`aria-label` state is inherited for role `button`
+The `aria-label` [state][] is [global][] and thus [inherited][] for all [semantic role][].
 
 ```html
 <div role="button" aria-label="OK"></div>
 ```
 
 #### Passed Example 5
 
-`aria-checked` state is required for role `aria-checkbox`
+The `aria-checked` [state][] is [required][] for the [semantic role][] `aria-checkbox`.
 
 ```html
 <div role="checkbox" aria-checked="false">My checkbox</div>
 ```
 
 #### Passed Example 6
 
-`aria-controls` property is supported for role `combobox`
+The `aria-controls` [property][] is [supported][] for the [semantic role][] `combobox`.
```

Comment:
The `aria-controls` [property][] is [required][] for the [semantic role][] `combobox`.
See Characteristics table under combobox: https://www.w3.org/TR/wai-aria-1.1/#combobox